src/entry.cpp

```diff
@@ -43,15 +43,11 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/entry.hpp"
 #include "libtorrent/hex.hpp"
 
-namespace
-{
-}
-
 namespace libtorrent
 {
 	namespace detail
 	{
-		TORRENT_EXPORT char const* integer_to_str(char* buf, int size
```

Comment:
I believe this is used from a public inlined function, isn't it? (in which case it does need to be exported)